### ISSUE TITLE
Help Maintaining this Page

### ISSUE THREAD

**Issue by daallgeier** [robchahin/sso-wall-of-shame#163]
Hi @robchahin,

I see that it sometimes takes you some time to update the page and pull in requests to update the page.  I'd be happy to help maintain this repo if you want to keep this page a bit more current and have it grow a bit faster.

What do you think?

**Comment by mbainter** [comment]
I am also pretty passionate about this problem and would likewise be glad to help manage the PRs and open issues.

**Comment by doransmestad** [comment]
@robchahin   I too would be happy to help keep the page up to date.

**Comment by doransmestad** [comment]
@daallgeier @mbainter   any word from @robchahin?    I'd love to see this repo and page get updated, but if robchahin isn't able to do so anymore I'm considering forking the repo and maintaining it myself to continue to push companies to offer SSO as part of their base offering.

**Comment by mbainter** [comment]
No, I don't believe I've seen anything.

**Comment by derekcwhite** [comment]
I, too, am eager to actively contribute to this project and even have a
new entry to shame. @robchahin/sso-wall-of-shame
***@***.***>, where
are you? Let us help!

Best regards,
Derek

**Comment by daallgeier** [comment]
Same for me. I haven't seen anything. I even tried Twitter and couldn't get
a reply.


Would really like to take this to the next level and ensure that we're
getting accurate and timely information for these companies.


It's a bit annoying to see that there are many long standing pull requests
that could be on the site easily. Having a handful of maintainers working
together on this would be great.


Shall I provide a (new) domain for your fork, Doran?

**Comment by derekcwhite** [comment]
Sign me up as maintainer here or in a fork, but can we give it until Friday
before registering a new domain? I've attempted reaching out to Rob via a
few additional avenues today, and since the sso.tax URL does have that
first mover advantage, I'd prefer to wait just a couple more days.

Best regards,
Derek

**Comment by doransmestad** [comment]
Yep, agreed.  Let's give it until Friday, just in case.  If we don't hear back, I'll start a fork, extend invites, and then we can discuss a domain.

**Comment by doransmestad** [comment]
Hi all-  as agreed, I waited until Friday before creating a fork:  https://github.com/doransmestad/sso-wall-of-shame.  I'm sad that we haven't heard from @robchahin, hope all is well with him.  We'll continue to carry his idea forward though!    I've invited you all as collaborators to the new repo.  Let's pick up conversation over there?

**Comment by Wild1145** [comment]
Would also be more than happy to donate some time to maintaining the site, it's been a very useful resource for me already so happy to help if I can :)

**Comment by mbainter** [comment]
@robchahin I saw there was a flurry of activity over the last day.  It probably doesn't make a lot of sense for us to have two projects covering the same ground.  Any chance we can work something out to make sure we're not duplicating effort and that the page doesn't get stale?

**Comment by daallgeier** [comment]
I'd suggest that we'll fork and merge from the current state and
continue maintaining independently.

**Comment by hotdang-ca** [comment]
Me too

**Comment by fleish** [comment]
Hi all, if I'm following the timeline here it looks like @doransmestad forked the project and spawned stopthesso.tax from it - but that github project was later archived on Jul 28, 2021. Since then, @robchahin committed and merged quite a lot into this github project (as well as a few commits from @NickBorgers & @nomis52). So a few questions:

- Is this project once again active and being maintained? I have a site to submit, but want to ensure it gets included in the correct location
- If this is the active project, does it make sense for @doransmestad to redirect stopthesso.tax back to sso.tax so we have a single source of truth?

Thanks in advance

**Comment by doransmestad** [comment]
@fleish   I can't speak for @robchahin  on his end of things, but unfortunately I got pulled away for other areas of work right after looking to invest more time in helping the cause here.  

If this repo is back and active again, great!   Alternatively, I'm happy to add additional maintainers in on the (now quite old) repository [here](https://github.com/stopthessotax/sso-wall-of-shame) as per the (rather prescient) [opening post](https://github.com/stopthessotax/sso-wall-of-shame/discussions/10) from @derekcwhite in the case we ourselves did not stay as active as intended.
